Client/src/modules/core/components/contact/Map.jsx

```
@@ -0,0 +1,16 @@
+const Map = () => {
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)